### DISCUSSION TITLE
Fans work with Turbo Button, rgb script does nothing

### DISCUSSION THREAD

**Discussion by DkmenchaR** [JafarAkhondali/acer-predator-turbo-and-rgb-keyboard-linux-module#169] [General]
Hi guys,
I tried to make a static light in a Triton 500 P515-52 but I just can't get rid of this ripple effect...
i can use shortcuts on keyboard, macros on the right side are not set right but it works.
i just wanted to make a static light in all keyboard, my model is a per key rgb
using arch 6.10 with wayland

**Comment by sirios-3D** [comment]
can not add service. after reboot turbo button not working. when i wright service codes it says 

"sent 2.676.224 bytes  received 355 bytes  5.353.158,00 bytes/sec
total size is 2.674.341  speedup is 1,00
[Create turbo-fan service]
Synchronizing state of turbo-fan.service with SysV service script with /usr/lib/systemd/systemd-sysv-install.
Executing: /usr/lib/systemd/systemd-sysv-install enable turbo-fan
update-rc.d: error: turbo-fan Default-Start contains no runlevels, aborting.
sirios@Predator-PH317-56:~/acer-predator-turbo-and-rgb-keyboard-linux-module$ 
"

**Reply by DkmenchaR** [reply]
Same with me, i just run the ./install.sh to just enable the turbo fan profile, every other script just doesnt work, maybe it needs a rework? i lost functionality after 6.11 kernel on arch, im no expert just put that script in chatgpt and he gave me a fix! Also im trying to get the ec firmware working in Linux, i found a way to capture packet trace from usb with windows in a VM, i just dont know what to do next, im no coder, i just like to fix stuff!

**Reply by DkmenchaR** [reply]
A pro coder might help make something, i know what I have to do, find the ec firmware port (usually is a usb connection under windows as ACPI/$DEV_ID$) even with wireshark, i can't undertand the info its there, i know where is the ec firmware in the board, i have an ch341a spi-programmer, maybe easily grab the PredatorSense profile while on windows, and flash it as "default"? Just to take rid of the ripple effect on keyboard as standard, at least you can use the laptop at night...
